monitor-scripts/network-detection.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+name="network-detection"
+service=$(cat /var/dashboard/services/$name | tr -d '\n')
+
+#检测网络链接畅通
+function network()
```

Comment:
@sunof-top `function check_server_status()` should be better than `function network()` , network() is a noun, we need a verb.